### PR TITLE
docs: rewrite gitlab security paragraph

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Rewrite paragraph

## Context

Use "we" and simpler words.
Make the call to action easier to read, and more inviting.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/gitlab-bot-security.md
Comment (HonkingGoose) [reply docs/usage/gitlab-bot-security.md:84]:
```suggestion
If you have any ideas, open a [discussion](https://github.com/renovatebot/renovate/discussions) and let us know!
```

**Comment by HonkingGoose** [comment]
This needs an approving review from a maintainer with `write` rights before it can get auto-merged. 😉

**Review by viceice — APPROVED** [review]

**Pull request merged**
